modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -54,37 +55,39 @@ When specifying a name, be sure to use the accepted syntax expression `^[a-zA-Z0
 
 <8> The number of the virtual functions (VF) to create for the SR-IOV physical network device. For an Intel network interface controller (NIC), the number of VFs cannot be larger than the total VFs supported by the device. For a Mellanox NIC, the number of VFs cannot be larger than `128`.
 
-<9> The NIC selector identifies the device for the Operator to configure. You do not have to specify values for all the parameters. It is recommended to identify the network device with enough precision to avoid selecting a device unintentionally.
+<9> Set `externallyManaged` to `true` to ensure some VFs are reserved for host managed systems. With the value set to `false` the SR-IOV Network Operator manages and configures all allocated VFs.
```

Comment:
please add a note here that the user need to create the vfs BEFORE they try to apply the policy if not the webhook will block the request

Reply:
also explain with false that the opeartor is the one that will create and even reset the virtual functions if needed.

so if the user use VFs on the host system he MUST create them manually and use the `externallyCreated: true` so the operator will not take any actions on the created vfs